sdk/storage/storage-blob/src/BlobBatchClient.ts

```diff
@@ -63,8 +65,20 @@ export declare type BlobBatchSetBlobsAccessTierResponse = BlobBatchSubmitBatchRe
 export class BlobBatchClient {
   private _serviceContext: Service;
 
-  constructor(service: Service) {
-    this._serviceContext = service;
+  /**
+   * Creates an instance of BlobBatchClient.
+   *
+   * @param {string} url A Client string pointing to Azure Storage blob service, such as
```

Comment:
>Client string [](start = 27, length = 13)

minor: Is the C by design to be uppercase? what about use url vs "Client string" directly?

Reply:
I prefer your suggestion to use url instead! :)